anyblok_pyramid/bloks/authorization/authorization.py

```diff
@@ -189,3 +191,105 @@ def check_validity(self):
             raise AuthorizationValidationException(
                 "Primary keys without model to apply in the authorization "
                 "(%s)" % self)
+
+    @classmethod
+    def ensure_authorizations_exists(
```

Comment:
probably ensure_authorization_exists would be better or ensure_exists as mentioned bellow ?

Reply:
done